### DISCUSSION TITLE
Do not place trees on water with scatter tool

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17668] [Feature requests] [answered]
![RCC Laydown 73 2022-07-31 23-15-47](https://user-images.githubusercontent.com/94884086/182128235-fb03462b-ca21-43db-90fb-aebdf31f86f0.png)
In my opinion, you should be allowed to place trees on water, but there should either be an optional checkbox, or just a preset so that when using the scatter tool, any object considered to be a plant will not be placed on top of water

**Comment by ocalhoun6** [comment, marked as answer]
I already suggested this in #10825 ... though that was back in the day before we were using discussions for feature requests.

Personally, I'd like to see a 3-way toggle between: on land only / on land and on water / on water only.
